Automattic/wp-calypso
# Marketplace Reviews: Show a Thank You section after updating a review

Fixes https://github.com/Automattic/dotcom-forge/issues/5026

## Proposed Changes

* Display a thank you section after updating a review
* Update the text area styling

![CleanShot 2024-01-10 at 15 23 59](https://github.com/Automattic/wp-calypso/assets/5039531/ca1454a4-68c9-4ffe-8b23-72cf352841f6)


## Testing Instructions
 Go to the plugin details page where you have reviews. Ex: `/plugins/woocommerce-bookings/`
* Scroll down and click on `Read all reviews`
* Update your review
*  You should see the Thank You section with confetti above the list of reviews
* Update again your review
* You should see the confetti being fired again

## Pre-merge Checklist
